src/Datasource/EntityInterface.php

```diff
@@ -159,91 +159,91 @@ public function setSource(string $alias);
+     * Returns an array with only the original fields
+     * stored in this entity, indexed by field name.
      *
-     * @param array $properties List of properties to be returned
+     * @param array $fields List of fields to be returned
      * @return array
      */
-    public function extractOriginalChanged(array $properties);
+    public function extractOriginalChanged(array $fields);
 
     /**
-     * Sets one or multiple properties to the specified value
+     * Sets one or multiple fields to the specified value
      *
-     * @param string|array $property the name of property to set or a list of
-     * properties with their respective values
-     * @param mixed $value The value to set to the property or an array if the
+     * @param string|array $field the name of field to set or a list of
+     * fields with their respective values
+     * @param mixed $value The value to set to the field or an array if the
      * first argument is also an array, in which case will be treated as $options
-     * @param array $options options to be used for setting the property. Allowed option
+     * @param array $options options to be used for setting the field. Allowed option
      * keys are `setter` and `guard`
      * @return \Cake\Datasource\EntityInterface
      */
-    public function set($property, $value = null, array $options = []);
+    public function set($field, $value = null, array $options = []);
 
     /**
-     * Returns the value of a property by name
+     * Returns the value of a field by name
      *
-     * @param string $property the name of the property to retrieve
+     * @param string $field the name of the field to retrieve
      * @return mixed
      */
-    public function &get($property);
+    public function &get($field);
 
     /**
-     * Returns whether this entity contains a property named $property
+     * Returns whether this entity contains a field named $field
      * regardless of if it is empty.
      *
-     * @param string|array $property The property to check.
+     * @param string|array $field The field to check.
      * @return bool
      */
-    public function has($property);
+    public function has($field);
 
     /**
-     * Removes a property or list of properties from this entity
+     * Removes a field or list of fields from this entity
      *
-     * @param string|array $property The property to unset.
+     * @param string|array $field The field to unset.
      * @return \Cake\Datasource\EntityInterface
      */
-    public function unsetProperty($property);
+    public function unsetField($field);
 
     /**
-     * Get the list of visible properties.
+     * Get the list of visible fields.
      *
-     * @return array A list of properties that are 'visible' in all representations.
+     * @return array A list of fields that are 'visible' in all representations.
      */
-    public function visibleProperties();
+    public function getVisible(): array;
```

Comment:
Creating backwards breaking changes for terminology cleanup seems harsh. Having both available lets us mark one as deprecated and document the preferred one.

Reply:
This is in this PR.

Reply:
My mistake I missed that.

Reply:
Is dropping it from the trait BC?